lib/ama_layout/navigation_helper.rb

```diff
@@ -0,0 +1,29 @@
+module AmaLayout
+  module NavigationHelper
+    def navigation
+      return AmaLayout::Navigation.non_member unless _member?(member?)
+      case
+      when _in_renewal_late?(status)
+        AmaLayout::Navigation.member_in_renewal_late
+      when _in_renewal?(in_renewal)
+        AmaLayout::Navigation.member_in_renewal
+      else
+        AmaLayout::Navigation.member
+      end
+    end
+
+    private
+
+    def _member?(member)
+      member
+    end
+
+    def _in_renewal?(in_renewal)
```

Comment:
@zoiec also in this PR. could these 3 methods just use member?, in_renewal and status == 'AL' without passing member?, in_renewal and status in as args?